src/modules/init2.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable import/order */
+// To organize imports in a better way
+import ApiService from '#src/services/api.service';
+import CleengService from '#src/services/cleeng.service';
+import AccountService from '#src/services/account.service';
+import CleengAccountService from '#src/services/cleeng.account.service';
+import CheckoutService from '#src/services/checkout.service';
+import CleengCheckoutService from '#src/services/cleeng.checkout.service';
+import SubscriptionService from '#src/services/subscription.service';
+import CleengSubscriptionService from '#src/services/cleeng.subscription.service';
+import InplayerAccountService from '#src/services/inplayer.account.service';
+import InplayerCheckoutService from '#src/services/inplayer.checkout.service';
+import SubscriptionJWService from '#src/services/inplayer.subscription.service';
+import WatchHistoryService from '#src/services/watchhistory.service';
+import ProfileService from '#src/services/profile.service';
+import EpgService from '#src/services/epg.service';
+import EntitlementService from '#src/services/entitlement.service';
+import FavoritesService from '#src/services/favorites.service';
+import ConfigService from '#src/services/config.service';
+import CleengProfileService from '#src/services/cleeng.profile.service';
+import InplayerProfileService from '#src/services/inplayer.profile.service';
+
+import ApiController from '#src/stores/ApiController';
+import WatchHistoryController from '#src/stores/WatchHistoryController';
+import CheckoutController from '#src/stores/CheckoutController';
+import AccountController from '#src/stores/AccountController';
+import EpgController from '#src/stores/EpgController';
+import EntitlementController from '#src/stores/EntitlementController';
+import ProfileController from '#src/stores/ProfileController';
+import FavoritesController from '#src/stores/FavoritesController';
+import AppController from '#src/stores/AppController2';
+import { INTEGRATION } from '#src/config';
+
+import { container } from './container';
+
+container.unbindAll();
```

Comment:
What if we make this a declarative method instead of just being inline and pass in a settings service as an argument?

Reply:
I was thinking about something similar. Do you suggest only wrapping the integrations bindings into a declarative method (otherwise, there would be no settings/config service)?

Perhaps passing the config or integration type is sufficient enough.

```ts
export const bindIntegration = (integrationType: string) => {
  if (integrationType === 'JWP') {
    // bind jwp services
  }

  if (integrationType === 'CLEENG') {
    // bind cleeng services
  }
};
```

Reply:
To me that sounds fine as well, which I think is just `init1` with some code split into another file, right?

Reply:
Yes, but I thought that was your original idea. Sounds to me that you have a different idea. Can you elaborate on that more, perhaps? 😄

Reply:
We will use a dynamic value 'INTEGRATION_TYPE'.